lib/manager/components/TransformationsViewer.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import Icon from '@conveyal/woonerf/components/icon'
+import React, { Component } from 'react'
+import { Col, ListGroup, Panel, Row, ListGroupItem, Badge } from 'react-bootstrap'
+
+import type {FeedVersion} from '../../types'
+
+type Props = {
+  version: FeedVersion,
+}
+
+export default class TransformationsViewer extends Component<Props> {
+  render () {
+    const {
+      version
+    } = this.props
+
+    if (version.feedTransformResult && version.feedTransformResult.tableTransformResults) {
+      const {tableTransformResults} = version.feedTransformResult
+      const transformContent = tableTransformResults.map(res => {
+        const badge = res.transformType === 'TABLE_MODIFIED'
+          ? <Badge>Table Modified</Badge>
+          : res.transformType === 'TABLE_ADDED'
+            ? <Badge>Table Added</Badge>
+            : res.transformType === 'TABLE_REPLACED'
+              ? <Badge>Table Replaced</Badge>
+              : res.transformType === 'TABLE_DELETED'
+                ? <Badge>Table Deleted</Badge>
+                : null
```

Comment:
Ooof use `switch` statement. You can even extract this to a function to avoid repeat `break` statements.

Reply:
copied this structure from somewhere else, I probably should get used to letting myself refactor as I go